pubspec.yaml

```diff
@@ -13,7 +13,7 @@ description: A mobile application for students at ACS UPB.
 # https://developer.apple.com/library/archive/documentation/General/Reference/InfoPlistKeyReference/Articles/CoreFoundationKeys.html
 #
 # ACS UPB Mobile uses semantic versioning. You can read more in the CONTRIBUTING.md file.
-version: 1.2.10+12
+version: 1.2.11+13
```

Comment:
Please modify the changelog files accordingly so we can trigger a new release.

Reply:
Or we can skip this release and wait until @andreicmirciu merges his feature this weekend, so we can include all changes in one release. WDYT?

Reply:
Sure, I think it's better to release it with @andreicmirciu 's feature